src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/MetadataTests/MetadataTests.cs

```diff
@@ -1,86 +1,8 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Collections.Generic;
-using System.Text.Json.Serialization;
-using System.Text.Json.Serialization.Tests;
-
 namespace System.Text.Json.Serialization.Tests
 {
-    public sealed class MetadataTests_Span : MetadataTests
-    {
-        public MetadataTests_Span() : base(JsonSerializerWrapper.SpanSerializer) { }
-    }
-
-    public sealed class MetadataTests_String : MetadataTests
-    {
-        public MetadataTests_String() : base(JsonSerializerWrapper.StringSerializer) { }
-    }
-
-    public sealed class MetadataTests_AsyncStream : MetadataTests
-    {
-        public MetadataTests_AsyncStream() : base(JsonSerializerWrapper.AsyncStreamSerializer) { }
```

Comment:
Because these are tests checking contract metadata and not serialization, targeting multiple serialization mediums is overkill. Leaving just async and string implementations.